etc/env/consensus_config.yaml

```diff
@@ -2,3 +2,10 @@ server_addr: '127.0.0.1:3054'
 public_addr: '127.0.0.1:3054'
 max_payload_size: 2500000
 gossip_dynamic_inbound_limit: 1
+genesis_spec:
+  chain_id: 1337
+  protocol_version: 1
+  validators:
+    - key: 'validator:public:bls12_381:b14e3126668ae79e689a2d65c56522889a3812ef5433097c33bd7af601b073dcdddf46e188883aa381725c49e08f90c705df1f78bf918e1978912cebeadff0d0084b1a4fe2ddee243e826348045f528803207f5de303c6a95bc1a701a190dbcf'
+      weight: 1
+  leader: 'validator:public:bls12_381:b14e3126668ae79e689a2d65c56522889a3812ef5433097c33bd7af601b073dcdddf46e188883aa381725c49e08f90c705df1f78bf918e1978912cebeadff0d0084b1a4fe2ddee243e826348045f528803207f5de303c6a95bc1a701a190dbcf'
```

Comment:
Could you please add comments like `# Should only be used in localhost environment` so that a) it doesn't get actually used in some envs (happened before with operator pk by accident) and b) there will be less questions from security if secret scan will recognize it one day?

Reply:
done